lambda_function/src/util.py

```diff
@@ -0,0 +1,29 @@
+"""
```

Comment:
this is a small amount of code. why not keep it in the lambda? that way it is clearer what is happening.